nginx/etc/confd/conf.d/nginx_site_redirect_to_https.conf.toml

```diff
@@ -1,6 +1,6 @@
 [template]
 src   = "nginx/site_redirect_to_https.conf.tmpl"
-dest  = "/etc/nginx/sites-available/default-20-site_redirect_to_https.conf"
+dest  = "/etc/nginx/sites-available/default-05-site_redirect_to_https.conf"
```

Comment:
It was moved from 05 to 20 in https://github.com/continuouspipe/dockerfiles/commit/48511980ea511737486ba76b782eb2fa8068bc0b, and it might still be needed to stop giving a 301 to a healthcheck.